.github/CODEOWNERS

```diff
@@ -127,7 +127,7 @@
 /pkg/orchestrator/                      @DataDog/container-app
 /pkg/network/                           @DataDog/networks
 /pkg/ebpf/                              @DataDog/networks
-/pkg/ebpf/bytecode/bindata/bindataPkgSecurity*      @DataDog/agent-security
+/pkg/ebpf/bytecode/bindata/bindataRuntimesecurity*      @DataDog/agent-security
```

Comment:
does this rely on all runtime security modules starting with name `runtime-security`? That seems a bit fragile to me but I'm not sure of a good way to get around it. And the problem was there before.

Reply:
It does. If they add more object files, we can try and make sure they get added to `CODEOWNERS` if they don't start with `runtime-security`.